.github/workflows/ci-master.yaml

```diff
@@ -24,6 +24,9 @@ jobs:
     runs-on: self-hosted
     steps:
     - uses: actions/checkout@v2
+    - uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: '5.0.201'
```

Comment:
Quick question - without this step, would your C# tests not run? Or does this just explicitly define the version needed?

Reply:
The runtime was defaulting to 3.1 and needed to be set explicitly for the tests to pass. Maybe the self-hosted instance simply needs the latest runtime?

Reply:
Thanks!